processing/dnsdumpster/install.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+from git import Repo
+
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "..", "..", "..")))
```

Comment:
```suggestion
sys.path.append(os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "..", "..", "..", "..")))
```

Reply:
My mistake; had been testing his as an added module library instead of from source of fame_modules